diff --git a/node_modules/vuepress-theme-vdoing/components/Home.vue b/node_modules/vuepress-theme-vdoing/components/Home.vue
index c22d81b..fae6391 100644
--- a/node_modules/vuepress-theme-vdoing/components/Home.vue
+++ b/node_modules/vuepress-theme-vdoing/components/Home.vue
@@ -24,13 +24,15 @@
           <p v-if="homeData.tagline" class="description">
             {{ homeData.tagline }}
           </p>
-          <p class="action" v-if="homeData.actionText && homeData.actionLink">
-            <NavLink class="action-button" :item="actionLink" />
+          <p class="action" v-if="hasActions">
+            <NavLink v-for="(action, index) in homeData.actions"  :key="index"
+              :class="action.actionClass || 'action-button'"
+              :item="{text: action.actionText, link: action.actionLink}" />
           </p>
         </header>
 
         <!-- PC端features块 s -->
-        <div class="features" v-if="hasFeatures && !isMQMobile">
+        <div class="features" v-if="hasFeatures" v-show="!isMQMobile">
           <div
             class="feature"
             v-for="(feature, index) in homeData.features"
@@ -162,7 +164,6 @@
     </MainLayout>
   </div>
 </template>
-
 <script>
 import NavLink from "@theme/components/NavLink";
 import BScroll from "@better-scroll/core"
@@ -182,7 +183,7 @@ BScroll.use(Slide)
 export default {
   data() {
     return {
-      isMQMobile: false,
+      isMQMobile: true,
 
       slide: null,
       currentPageIndex: 0,
@@ -203,6 +204,9 @@ export default {
     hasFeatures() {
       return !!(this.homeData.features && this.homeData.features.length)
     },
+    hasActions() {
+      return !!(this.homeData.actions && this.homeData.actions.length)
+    },
     homeSidebarB() {
       const { htmlModules } = this.$themeConfig
       return htmlModules ? htmlModules.homeSidebarB : ''
@@ -233,12 +237,6 @@ export default {
         return `background: url(${this.$withBase(bannerBg)}) center center / cover no-repeat`
       }
 
-    },
-    actionLink() {
-      return {
-        link: this.homeData.actionLink,
-        text: this.homeData.actionText
-      };
     }
   },
   components: { NavLink, MainLayout, PostList, UpdateArticle, BloggerBar, CategoriesBar, TagsBar, Pagination },
